pkg/api/mlflow/service/artifact/validators.go

```diff
@@ -32,3 +32,26 @@ func ValidateListArtifactsRequest(req *request.ListArtifactsRequest) error {
 	}
 	return nil
 }
+
+// ValidateGetArtifactRequest validates `GET /get-artifact` request.
```

Comment:
please update `GET /get-artifact` according to new route.

Reply:
will do